pkg/util/utils.go

```diff
@@ -69,6 +69,32 @@ func StringInSlice(s string, sl []string) bool {
 	return false
 }
 
+// ParseChanges returns key, value(s) pair for given option.
+func ParseChanges(option string) (key string, vals []string) {
```

Comment:
Can we add unit tests for it? Parts of it is already tested in the integration but we should throw more input at it. We've regressed several times on parsing functions, so I prefer to have some extra security net.

Reply:
Thanks for the great comments all around the code!